docs/pages/inboxes/build-inbox.mdx

```diff
@@ -26,202 +26,204 @@ Learn to build a chat inbox with the help of [XMTP.chat](https://xmtp.chat/), an
 
 ## Create or build a client
 
-### Create an account signer
+### Create an account signer 
```

Comment:
this section needs updates per:

`SigningKey` now supports only one sign method: `sign(signatureText: String): ByteArray`. It no longer supports the following sign methods:

- `signSCW(signatureText: String): ByteArray`
- `sign(message: String): Proto.Signature`
- `sign(digest: ByteArray): Proto.Signature`

Reply:
This is universal across all sdks now the signers should all align